documentation/homework/Homework4-MobX/my-app/src/todos/CreateTodo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {observer,inject} from 'mobx-react'
+
+@inject('todoStore')
+@observer
+export default class CreateTodo extends React.Component {
+    addTodo = e => {
+        if (e.which === 13) {
+            this.props.todoStore.createTodo(e.target.value.trim())
+            e.target.value = ''
+        }
+    }
+
+    render() {
+        return ( <input style = {
```

Comment:
No un-controlled components.
you are not handeling the changes which means it's not working 😶
[For more detailes...](https://reactjs.org/docs/forms.html)